### PR TITLE
Web post to accompany the initial release

### PR DESCRIPTION
**Pull request by lbulej**
This is a draft of the post to accompany the initial release of the suite. Note that I'm neither sure whether we can link to the PLDI paper prior to publication nor what the URL will be. 

Comments and/or edits are welcome :-)

### CODE REVIEW

**Comment by ceresek** [comment]
I think you've won the bid for our official PR contact :)

File: website/_drafts/2019-05-06-renaissance-0-9-0.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: mainpost
+projectname: Renaissance Suite
+title:  "Renaissance 0.9 Released"
+author: TODO
```
Comment (farquet) [inline website/_drafts/2019-05-06-renaissance-0-9-0.md]: Should be addressed before merging.

Comment (lbulej) [reply website/_drafts/2019-05-06-renaissance-0-9-0.md]: Ooops... Metadata, metadata....

**Comment by axel22** [comment]
Looks great!

**Comment by lbulej** [comment]
The post has been made live on the site.

**Pull request merged**